m4_sessions/1808-inning/surprise_of_happiness.md

```diff
@@ -1,18 +1,30 @@
 ## Group Members:
+Haley Mesander
+Sabrina Robinson
+Tory Dannenberg
 
-
-
-## Name of Session: 
+## Name of Session: Surprise of Happiness
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+     What makes you happy? That's what you think...
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+     More useful ways to benefit your pom (fun exercise challenges, going outside, listening to a song). 
+
+
 
 ## Session Outline 
+     10 min for intros
+     5 min for pairing
+     5 min quiz
+     5 min to discuss quiz that they relate with
+     5 min to discuss quiz that they didn't relate with
+     10 min for video
+     10 min pom
```

Comment:
I was confused about a 10 min pom, but now I think I get it.  Are you doing a 10 min pom where they practice making their poms better?

Reply:
Yes, we will give them the first few minutes to go to the bathroom, etc and then will spend the rest practicing new techniques to make poms better.